lib/rspec/support/with_keywords_when_needed.rb

```diff
@@ -0,0 +1,33 @@
+RSpec::Support.require_rspec_support("method_signature")
```

Comment:
Isn't `method_signature_verifier` that you want to require?

Reply:
It is indeed, pushed up for the expectations build without checking :joy: